update.py

```diff
@@ -84,20 +87,44 @@ def fetchparameters(site=args.site):
 
 
 
+def build_one(wiki):
+    '''build one wiki'''
+    print('make and clean: %s' % wiki)
+    subprocess.check_call(["nice", "make", "clean"], cwd=wiki)
+    subprocess.check_call(["nice", "make", "html"], cwd=wiki)
 
 def sphinx_make(site):
     """
     Calls 'make html' to build each site
     """
-
-    for wiki in ALL_WIKIS:
+    done = set()
+    wikis = set(ALL_WIKIS[:])
+    num_procs = 0
+    procs = []
+
+    while len(done) != len(wikis):
+        wiki = list(wikis.difference(done))[0]
+        done.add(wiki)
         if site=='common':
             continue
         if not site==None and not site==wiki:
             continue
         print('make and clean: %s' % wiki)
```

Comment:
This is repeated now, we should remove it here.

Reply:
done